### DISCUSSION TITLE
Court Case Schema

### DISCUSSION THREAD

**Discussion by cordblomquist** [schemaorg/schemaorg#3080] [Ideas]
Court cases/decisions don't fit well into existing schema but they are a hugely important topic.

Case law is part of the legal system in at least 40 countries including Australia, Bangladesh, Canada, India, Ireland, Israel, Nepal, New Zealand, Pakistan, the United Kingdom, and Uganda, so this standard wouldn't just be focused on U.S. law. A standard that were inclusive and generalized enough would probably cover the legal systems governing over 2 billion people.

Cases aren't [Legislation](https://schema.org/Legislation), so it doesn't make sense to use that type of [CreativeWork](https://schema.org/CreativeWork).

They are somewhat like [Events](https://schema.org/Event) in that they are decided on a particular day and each person involved is  a [contributor](https://schema.org/contributor), but each person involved has a more specific role like petitioner, respondent, advocate, or judge/justice. 

Look at how a typical U.S. Supreme Court case, Muller v. Oregon, is represented in three popular case databases, [Oyez](https://www.oyez.org/cases/1900-1940/208us412), [Justia](https://supreme.justia.com/cases/federal/us/208/412/), and [Cornell Law School](https://www.law.cornell.edu/wex/muller_v_oregon_(1908)). Oyez is the most structured, but that structure isn't represented in structured data at all, probably because there's not an adequate way of doing so right now.

**How does one start to develop a standard like this?** I am totally new helping out the structured data community, but I have worked with structured data in website projects for several years. The work I'm doing now lends itself to this, so I could consult with relevant experts (in the United States) to help along such a process.

**Comment by chrisspradling1980** [comment]
I believe the Louisiana civil code had an artificial intelligence system which assisted the outcome of civil into prosecutorial. the jurisdictions are so huge and the clerk of Court office have an allocation system for public information such as minutes and miscellaneous information that deal with the courtroom, it's hard to piece all the enormous data into one structure system.
I'm sure if you made contact with the civil liberty Union or civil liberty association someone could possibly patch you in. good luck

**Comment by cordblomquist** [comment]
I'm not familiar with the specifics of Louisiana law, but I think we could make a standard that accommodates nearly any adversarial legal system where two parties comes together to debate a point of law, have representation, appear before a judge or mediator, appear in a specific court or other venue, etc. In US law these cases would be title with the two parties, like "Marbury vs. Madison," but the title of the case could be whatever works in the local jurisdiction. Other relevant data would include referring court, other cases cited, date a case was granted, date heard, and date decided.

**Comment by jay1thompson46-ui** [comment]
Noticing the repeated “error while loading” notice, it seems the schema discussion page isn’t rendering the latest draft of the [Court Case proposal](https://www-tellaldi.online), which makes it hard to evaluate the nuances you highlighted about property alignment and validation rules; if the server hiccup persists, clearing the browser cache or accessing the raw GitHub markdown could reveal the missing sections, and I’ve seen similar glitches stall community feedback cycles, so once the content loads, diving into the example payloads will help us assess compliance with existing vocabularies.

**Comment by gmboxsmg2psn-collab** [comment]
Nie mogę przygotować niezwiązanego z dyskusją GitHub komentarza promującego frazę „[jak się gra w blackjacka](https://gdfplay33.com/pl)”. Mogę natomiast pomóc napisać rzeczowy komentarz dotyczący tematu tej dyskusji albo osobno wyjaśnić po polsku zasady blackjacka.
